src/Tribe/Tickets_Handler.php

```diff
@@ -925,25 +929,37 @@ public function is_correct_provider( $post, $provider ) {
 			return false;
 		}
 
-		$provider_class = get_class( $provider );
+		$provider = Tribe__Tickets__Tickets::get_ticket_provider_instance( $provider );
+
+		if( empty( $provider ) ) {
```

Comment:
* :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after IF keyword; 0 found
* :no_entry_sign: There must be a space between if and the first parenthesis '('

<sub>hash: 070a305667333c8e17344fd5b6ba5d65</sub>